db/1.9/modules/ROOT/edits/database-connector-studio.adoc

```diff
@@ -0,0 +1,135 @@
+
+Anypoint Studio (Studio) editors help you design and update your Mule applications, properties, and configuration files.
+
+To add and configure a connector in Studio:
+
+. <<create-mule-project,Create a Mule project>>.
+. <<add-connector-to-project,Add the connector to your Mule project>>.
+. <<configure-input-source,Configure a source for the connector's flow>>.
+. <<add-connector-operation,Add a connector operation to the flow>>.
+. <<configure-global-element,Configure a global element for the connector>>.
+
+When you run the connector, you can view the app log to check for problems, as described in <<view-app-log,View the App Log>>.
+
+
+If you are new to configuring connectors in Studio, see xref:connectors::introduction/intro-config-use-studio.adoc[Use Studio to Configure a Connector]. If, after reading this topic, you need additional information about the connector fields, see the xref:database-documentation.adoc[Database Connector Reference].
+
+[[create-mule-project]]
+== Create a Mule Project
+
+In Studio, create a new Mule project in which to add and configure the connector:
+
+. In Studio, select *File > New > Mule Project*.
+. Enter a name for your Mule project and click *Finish*.
+
+[[add-connector-to-project]]
+== Add the Connector to Your Mule Project
+
+Add Database Connector to your Mule project to automatically populate the XML code with the connector's namespace and schema location and to add the required dependencies to the project's `pom.xml` file:
+
+. In the *Mule Palette* view, click *(X) Search in Exchange*.
+. In the *Add Dependencies to Project* window, type `database` in the search field.
+. Click *Database Connector* in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+[[configure-input-source]]
+== Configure a Source
+
+A source initiates a flow when a specified condition is met.
+You can configure one of these input sources to use with Database Connector:
+
+* *Database > On Table Row* +
+Initiates a flow by selecting from a table at a regular interval and generates one message per obtained row
+* *HTTP > Listener* +
+Initiates a flow each time it receives a request on the configured host and port
+* *Scheduler* +
+Initiates a flow when a time-based condition is met
+
+For example, to configure an *On Table Row* source, follow these steps:
+
+. In the *Mule Palette* view, select *Database > On Table Row*.
+. Drag *On Table Row* to the Studio canvas.
+. On the *On Table Row* configuration screen, optionally change the value of the *Display Name* field.
+. Click the plus sign (*+*) next to the *Connector configuration* field to configure a global element that can be used by all instances of the source in the app.
+. On the *Database Config* window, for *Connection* select any of the connection types to provide to this configuration:
```

Comment:
ditto

Reply:
Again, I would like to see a clearer wording for navigation to a lower-level GUI control, in this case, something like either of...:
```suggestion
. In *Connection* [section? box? menu? list?...] of the *Database Config* window, select any of the connection types to provide to this configuration:
```
OR
```suggestion
. In *Database Config* > *Connection*, select any of the connection types to provide to this configuration:
```

Reply:
GLOBAL for steps worded this way throughout this set of topics.